third_party/CMakeLists.txt

```diff
@@ -13,11 +13,9 @@ if(THRIVE_AVX)
 
   # Not directly AVX instructions, but are about as new so are only on when AVX is on
   set(USE_F16C ON)
+  set(USE_FMADD ON)
   set(USE_LZCNT ON)
   set(USE_TZCNT ON)
-
-  # this is off anyway as this causes indeterminism in the simulation
-  set(USE_FMADD OFF)
```

Comment:
Why was this toggled? I'd prefer to keep the physics deterministic so that if we ever add multiplayer we don't have a ton of game state synchronization issues with clientside prediction.